integration/mediation-tests/tests-patches/src/test/resources/testng.xml

```diff
@@ -148,10 +148,12 @@
 
     <test name="Carbon-app-deployment-test" preserve-order="true" verbose="2">
         <classes>
-            <class name="org.wso2.carbon.esb.car.deployment.test.ESBJAVA3611EndpointTestCase"/>
+            <!-- Disable testcases as checking the service deployment is not supported
+            https://github.com/wso2/micro-integrator/issues/366 -->
+            <!--<class name="org.wso2.carbon.esb.car.deployment.test.ESBJAVA3611EndpointTestCase"/>-->
             <class name="org.wso2.carbon.esb.car.deployment.test.ESBJAVA3546CAppProxyOnLoadTestCase"/>
-            <class name="org.wso2.carbon.esb.car.deployment.test.ESBJAVA4116CAppArtifactDeploymentFailureDueToNameTestCase"/>
-            <class name="org.wso2.carbon.esb.car.deployment.test.CarbonApplicationReDeploymentTestCase"/>
+            <!--<class name="org.wso2.carbon.esb.car.deployment.test.ESBJAVA4116CAppArtifactDeploymentFailureDueToNameTestCase"/>-->
+            <!--<class name="org.wso2.carbon.esb.car.deployment.test.CarbonApplicationReDeploymentTestCase"/>-->
```

Comment:
Shall we clean up this rather than commenting 
and let's delete the classes as well

Reply:
This is to exclude the testcase as this functionality is not there at the moment. Once we resolve https://github.com/wso2/micro-integrator/issues/366 we can enable this and test this testcase. WDYT?